metadata.rb

```diff
@@ -5,6 +5,7 @@
 version           "0.0.2"
 
 depends "rbenv"
+depends "postfix"
```

Comment:
Since Chef 11, depends is the only way that means the cookbook's attributes are loaded. If it's recommends, the attributes wont be loaded for the `include_recipe 'postfix'` bit, so postfix cookbook default recipe will run without it's attributes.

The negatives of `depends` only means the cookbook will be installed by the berks/librarian tool, not that it will be added to the run list.

You'll see the same with apt and yum cookbooks for example